src/cbmc/cbmc_parse_options.cpp

```diff
@@ -111,6 +130,8 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
   if(cmdline.isset("paths"))
     options.set_option("paths", true);
 
+  cbmc_parse_optionst::set_default_options(options);
+
```

Comment:
Nit pick: may I suggest to move this up by three lines to be the very first step before doing `if(cmdline.isset...`?